src/main/java/me/timmy109/betterreports/commands/AdminCommand.java

```diff
@@ -42,53 +43,46 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 		List<String> adminHelp = ArrayUtils.getAdminHelpList();
 		List<String> playerHelp = ArrayUtils.getPlayerHelpList();
 
-
 		if (args.length == 0) {
 			if (sender.hasPermission("betterreports.admin")) {
-				for (String s : adminHelp) {
-					sender.sendMessage(Common.color(s));
-				}
+				adminHelp.forEach(s -> sender.sendMessage(Common.color(s)));
 				return true;
 			}
 
-			for (String s : playerHelp) {
-				sender.sendMessage(Common.color(s));
-			}
+			playerHelp.forEach(s -> sender.sendMessage(Common.color(s)));
+			return true;
+		}
+
+		if (args.length > 1) {
+			sender.sendMessage(Common.color("&cUnknown Command"));
 			return true;
 		}
 
 		switch (args[0]) {
-			case "debug": {
-				if (sender.hasPermission("betterreports.admin")) {
-					for (String s : debug) {
-						sender.sendMessage(Common.color(s));
-					}
-				} else {
+			case "debug":
+				if (!sender.hasPermission("betterreports.admin")) {
 					sender.sendMessage(Common.color("&cYou do not have permission to execute this command!"));
+					return true;
 				}
-				break;
-			}
-			case "reload": {
-				if (sender.hasPermission("betterreports.reload")) {
-					try {
-						BetterReports.getInstance().reloadConfig();
-						for (String s : reload) {
-							sender.sendMessage(Common.color(s));
-						}
-					} catch (Exception ex) {
-						sender.sendMessage(Common.color("&cThere was an error reloading the config. Check console for more details."));
-					}
-				} else {
+				debug.forEach(s -> sender.sendMessage(Common.color(s)));
+				return true;
+
+			case "reload":
+				if (!sender.hasPermission("betterreports.reload")) {
 					sender.sendMessage(Common.color("&cYou do not have permission to execute this command!"));
+					return true;
 				}
-				break;
-			}
+				try {
+					BetterReports.getInstance().reloadConfig();
+					reload.forEach(s -> sender.sendMessage(Common.color(s)));
+				} catch (Exception ex) {
+					ex.printStackTrace();
+					sender.sendMessage(Common.color("&cThere was an error reloading the config. Check console for more details."));
+				}
+				return true;
 		}
 
-		if (args.length > 1) {
-			sender.sendMessage(Common.color("&cUnknown Command"));
-			return true;
-		}
+		sender.sendMessage(Common.color("&cUnknown Command"));
```

Comment:
There does not need to be an Unknown command message here, add a "default" statement to the switch block.